homeassistant/components/shelly/sensor.py

```diff
@@ -358,7 +358,7 @@ class RestSensorDescription(RestEntityDescription, SensorEntityDescription):
         use_polling_coordinator=True,
     ),
     "temperature_0": RpcSensorDescription(
-        key="temperature:0",
+        key="temperature",
```

Comment:
We use this to generate the `unique_id`, but this is not a breaking change since the final key will get appended with `:0`, see https://github.com/home-assistant/core/blob/35a47d83a48fc410f51fd5297f93e95291cd3dff/homeassistant/components/shelly/utils.py#L297